lib/resources/orders.js

```diff
@@ -43,11 +43,11 @@ module.exports = function (api) {
     },
 
     create(params = {}, callback) {
-      let { amount, currency, receipt, payment_capture, notes,
+      let { amount, currency, receipt, payment_capture, notes, method,
             ...otherParams } = params
       currency = currency || 'INR'
 
-      if (!amount) {
+      if (!(amount || (method === 'emandate' && amount === 0))) {
```

Comment:
```suggestion
      if (!amount || (amount === 0 && method !== 'emandate')) {
```

Reply:
@geekguy Can you make this change and rebuild it? Looks good to go.

Reply:
@geekguy bump

Reply:
@umanghome - I don't think that will work.

when we pass `amount = 0`, `!amount` will become `true` and throw an error. 

No?

Reply:
Oh, yes.